components/script/dom/htmlareaelement.rs

```diff
@@ -35,8 +189,28 @@ impl HTMLAreaElement {
                prefix: Option<DOMString>,
                document: &Document) -> Root<HTMLAreaElement> {
         Node::reflect_node(box HTMLAreaElement::new_inherited(local_name, prefix, document),
-                           document,
-                           HTMLAreaElementBinding::Wrap)
+               document,
+               HTMLAreaElementBinding::Wrap)
+    }
+
+    #[warn(dead_code)]
+    fn get_shape_from_coords(&self) -> Option<Area> {
+       let elem = self.upcast::<Element>();
+       let shp: Shape;
+       let shape = elem.get_string_attribute(&LocalName::from("shape"));
```

Comment:
You should be able to add `shape` to https://dxr.mozilla.org/servo/source/components/atoms/static_atoms.txt and use `atom!("shape")` instead.